contracts/community-tokens/contracts/BaseToken.sol

```diff
@@ -116,6 +129,15 @@ contract CollectibleV1 is
         return baseTokenURI;
     }
 
+    function _mintTo(address[] memory addresses) internal {
+        // We cannot just use totalSupply() to create the new tokenId because tokens
+        // can be burned so we use a separate counter.
+        for (uint256 i = 0; i < addresses.length; i++) {
+            _safeMint(addresses[i], _tokenIdTracker.current(), "");
+            _tokenIdTracker.increment();
+        }
```

Comment:
One thing we might want to consider is figuring out beforehand how man `addresses` we'll likely expect.
If this will be done for hundreds of addresses, there are more gas efficient options to do this.

Maybe something to keep in mind for a v2 of this.